static/templates/gear_menu.hbs

```diff
@@ -8,6 +8,30 @@
             view is a Bootstrap nav tab, even though we don't show the tab
             anymore --}}
             <li class="invisible" style="display:none;" role="presentation"><a href="#message_feed_container" data-toggle="tab"></a></li>
+            <li class="org-info org-name">{{realm_name}}</li>
+            <li class="org-info org-url">{{realm_uri}}</li>
+            <li class="org-info org-plan hidden-for-spectators">
+                {{#if is_plan_limited }}
+                    <a href="/plans" target="_blank" rel="noopener noreferrer" role="menuitem">Zulip Cloud Free</a>
+                {{else if is_plan_standard}}
+                    <a href="/plans" target="_blank" rel="noopener noreferrer" role="menuitem">Zulip Cloud Standard</a>
+                {{else if is_plan_standard_sponsored_for_free}}
+                    <a href="/plans" target="_blank" rel="noopener noreferrer" role="menuitem">Zulip Cloud Standard,<br />sponsored for free</a>
+                {{else if is_self_hosted}}
+                    Version {{zulip_version}}
```

Comment:
Certainly this one should be tagged for translation.

Reply:
Done.